packages/editor/src/utils/analytics.js

```diff
@@ -44,14 +44,27 @@ export function logEvent(eventType, eventProperties) {
   return amplitude.getInstance().logEvent(eventType, eventProperties);
 }
 
-export const AnalyticsProvider = ({children}) => (
-    <AmplitudeProvider
-        amplitudeInstance={amplitude.getInstance()}
-        apiKey={AMPLITUDE_KEY}
-    >
-        {children}
-    </AmplitudeProvider>
-);
+export const AnalyticsProvider = ({children}) => {
```

Comment:
Why this has changed in this PR?

Reply:
Forgot to remove this. But now it is needed as I also removed the AMPLITUDE variable, as we don't need to track such info in local environment.